tests/Keboola/S3ExtractorTest/Functional/ExceptionsFunctionalTest.php

```diff
@@ -79,6 +79,26 @@ public function testInvalidKey()
         $application->actionRun($this->path);
     }
 
+    public function testMissingWildcardOrPathFile()
+    {
+        $this->expectException(Exception::class);
+        $this->expectExceptionMessage('Enter a wildcard flag or full path to the file.');
```

Comment:
pro poradek bych radsi aby ten expect byl az pred tim mistem, kde ta vyjimka opravdu nastane - actionRun

Reply:
S tím souhlasím, držel jsem se současného CS testů. Prohodil jsem to :)